src/styles/Components/src/tokens-card.css

```diff
@@ -0,0 +1,34 @@
+:root {
+    --denhaag-card-background-color: var(--denhaag-white);
+
+    --denhaag-card-border-radius: 3px;
+    --denhaag-card-border-width: 1px;
+    --denhaag-card-border-color: var(--denhaag-grey-2);
+
+    --denhaag-card-height: 181px;
+    --denhaag-card-width: 352px;
+    --denhaag-card-padding-block: 24px;
+    --denhaag-card-padding-inline: 24px;
+
+    --denhaag-card-subtitle-color: var(--denhaag-grey-4);
+    --denhaag-card-subtitle-font-size: var(--denhaag-font-size-base);
+    --denhaag-card-subtitle-line-height: 24px;
+    --denhaag-card-subtitle-font-weight: 500;
+    --denhaag-card-subtitle-padding: 0;
```

Comment:
Could we specify which axis for padding this should apply to? (Inline/Block)

Reply:
Right now it applies to all padding of the subtitle i'm not sure if we want to have individual tokens for all of the padding of the subtitle. @mjcarsjens what do you think?

Reply:
I'm fine with splitting this up into `inline` and `block`, feel like there's a valid usecase for that. You can always just use the same setup as above for `--denhaag-card-wrapper-padding`